### DISCUSSION TITLE
Accessing color palette shades in theme colors

### DISCUSSION THREAD

**Discussion by nate-fur** [saadeghi/daisyui#1411] [Q&A] [answered]
Hello. Am wondering how I can access different shades of my color palette inside a custom daisyUI theme.

I have overridden Tailwind's default color palette as such: 

![Screenshot 2022-12-02 at 11 48 18 AM](https://user-images.githubusercontent.com/37522274/205374345-bfdec934-e76a-4cce-8b79-0c26a82fdfff.png)

I want to set orange-500 to be the primary color in my new theme, but am getting the following error `Error: Unable to parse color from string: orange-500`. Is there any syntax that allows for the theme to accept different _shades_ of a color that is defined in the color palette?

![Screenshot 2022-12-02 at 11 49 51 AM](https://user-images.githubusercontent.com/37522274/205374559-2ab0586e-9c69-47d3-869e-ece90adbe507.png)

Thanks!

**Comment by saadeghi** [comment, marked as answer]
One way would be putting the TailwindCSS `theme` object in a variable so we can call it by name: 
```js
const tailwindThemeObject = {
  colors: {
    orange: {
      500: "#F18B79",
      //...
    }
  }
}

module.exports = {
  plugins: [
    require('daisyui'),
  ],
  theme: tailwindThemeObject,
  daisyui: {
    themes: [
      {
        light: {
          ...require("daisyui/src/colors/themes")["[data-theme=light]"],
          primary: tailwindThemeObject.colors.orange[500],
        },
      },
    ],
  },
}
```
